conjur/util.py

```diff
@@ -21,17 +21,17 @@
 
 # 2 vs 3 urlencode moved...
 try:
-    from urllib import urlencode, quote
+    from urllib import quote
 except:
-    from urllib.parse import urlencode, quote
+    from urllib.parse import quote
 
 
 def urlescape(s):
     return quote(s, '')
 
 
 def authzid(obj, kind, with_account=True):
-    if isinstance(obj, basestring):
+    if isinstance(obj, (str, unicode)):  # noqa F821 (flake8 doesn't know about unicode)
```

Comment:
Neither works with flake8.  Not sure why the change.  It's equivalent I believe.